app/assets/javascripts/components/overview/notify_instructors_button.jsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { initiateConfirm } from '../../actions/confirm_actions.js';
+import TextAreaInput from '@components/common/text_area_input.jsx';
+import { useDispatch } from 'react-redux';
+import Loading from '@components/common/loading.jsx';
+import API from '../../utils/api.js';
+import { ADD_NOTIFICATION } from '../../constants/notifications.js';
+import TextInput from '@components/common/text_input.jsx';
+
+NotifyInstructorsButton.propTypes = {
+  courseId: PropTypes.number.isRequired,
+  courseTitle: PropTypes.string.isRequired,
+};
+
+const initialState = {
+  subject: '',
+  message: '',
+  status: 'DEFAULT', // DEFAULT, PENDING, FAILED
+  error: null,
+};
+
+export default function NotifyInstructorsButton(props) {
```

Comment:
For style consistency, I'd prefer to use ES6 arrow functions for functional components, rather than traditional function syntax.